themes/ddbasic/template.node.php

```diff
@@ -194,10 +194,18 @@ function ddbasic_preprocess__node__ding_event(&$variables) {
 
       // Date.
       if (!empty($date)) {
-        $start_date = strtotime($date[0]['value']);
-        $end_date = strtotime($date[0]['value2']);
-
-        $variables['event_date'] = format_date($start_date, 'ding_date_only_version2');
+        // @see http://www.drupalcontrib.org/api/drupal/contributions%21date%21date_api%21date_api.module/class/DateObject/7
+        // Create a dateObject from startdate, set base timezone to UTC
+        $date_start = new DateObject($date[0]['value'], new DateTimeZone($date[0]['timezone_db']));
+        // Set timezone to local timezone
+        $date_start->setTimezone(new DateTimeZone($date[0]['timezone']));
```

Comment:
Why not do?:
$date_start = new DateObject($date[0]['value'], new DateTimeZone($date[0]['timezone']));
Or?:
$date_start = new DateObject($date[0]['value']);
$date_start->setTimezone(new DateTimeZone($date[0]['timezone']));